pkg/component/component.go

```diff
@@ -446,3 +446,17 @@ func GetComponentDesc(client *occlient.Client, currentComponent string, currentA
 
 	return componentType, path, componentURL, appStore, nil
 }
+
+// Get Component logs
+func GetLogs(client *occlient.Client, applicationName string) (string, error) {
```

Comment:
So for io.Writer, should we have this functionality higher-up, in the `cmd` package instead? At the moment, we have the "color" functionlity (using `color.Output`) within component.go rather than up in `cmd`. So perhaps I should move both.

Reply:
Not sure if I understand that :-(

There are functions that will print output in two colors.
For example `component.Watch`.
It is called from `cmd` package and it takes `io.Writer` as argument.
It writes to `io.Writer` messages in white and also in yellow. So the color can't be set in `cmd` package.